### ISSUE TITLE
Do these Codes and read Readme File 

### ISSUE THREAD

**Issue by Vishal-Aggarwal0305** [Vishal-Aggarwal0305/DSA#72]
Only these codes will be accepted now:

https://medium.com/techie-delight/500-data-structures-and-algorithms-practice-problems-35afe8a1e222

**Comment by georgian46006** [comment]
ok i will take care of it.

**Comment by HackX-OP** [comment]
Can you assign this under hacktober fest I would like to add code for merging two sorted arrays.

**Comment by Vishal-Aggarwal0305** [comment]
> Can you assign this under hacktober fest I would like to add code for merging two sorted arrays.

yes read readme file and follow the naming pattern  category_codenumber_codenumber.cpp

**Comment by sandesh31122000** [comment]
can i work on it?

**Comment by saloni-jaiswal-dev** [comment]
Hey, please assign this issue to me.

**Comment by ghost** [comment]
hey can i solve these ?
